ifbcat_api/model/event.py

```diff
@@ -138,7 +144,15 @@ class EventAccessibilityType(models.TextChoices):
         blank=True,
         help_text="URL of event homepage.",
     )
-    onlineOnly = models.BooleanField(null=True, blank=True, help_text="Whether the event is hosted online only.")
+    is_draft = models.BooleanField(null=True, blank=True, help_text="Mention if the event is a draft")
```

Comment:
help_text should be more generic : training will get an help text with the term event.

Proposition: `When True, some require attributes become optional, and the item is filtered out in the API by default`

We should also filter out the is_draft elements from the API I think

Reply:
ok got it 👍🏾